fdbserver/storageserver.actor.cpp

```diff
@@ -9264,6 +9264,9 @@ ACTOR Future<Void> fetchKeys(StorageServer* data, AddingShard* shard) {
 					const double ingestDuration = g_network->timer() - ingestStartTime;
 					data->counters.ingestDurationLatencySample->addMeasurement(ingestDuration);
 
+					// Compact the range after ingestion to avoid accumulating TODO compaction
```

Comment:
You want to remove the TODO here?

Reply:
Oh, the TODO here means that compaction that will be triggered in the future. I can change.